libbeat/outputs/redis/docs/redis.asciidoc

```diff
@@ -48,6 +48,13 @@ distributed to the servers in the list. If one server becomes unreachable, the e
 distributed to the reachable servers only. You can define each Redis server by specifying
 `HOST` or `HOST:PORT`. For example: `"192.15.3.2"` or `"test.redis.io:12345"`. If you
 don't specify a port number, the value configured by `port` is used.
+Each Redis server can be defined as a `IP:PORT` pair or as a `URL`. For
```

Comment:
I'd change this to say: 

```
Define each Redis server as an `IP:PORT` pair or a `URL`.
```

Note that pronunciation determines whether we use "a" or "an".

Reply:
Changing to: `Configure each Redis server with an `IP:PORT` pair or with a `URL`.`
 
I get the `an 'IP:PORT'`, but why it says `a 'URL'` is beyond me :P

Reply:
The English language likes to confuse people. :-) In this case, the pronunciation "you-are-ell" begins with a consonant sound, so "a" is the correct article to use. Crazy huh?

Reply:
TIL, thanks :)